platform/android/export/export.cpp

```diff
@@ -1978,6 +1978,7 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 				for (int i = 0; i < enabled_abis.size(); ++i) {
 					if (file.begins_with("lib/" + enabled_abis[i] + "/")) {
 						enabled = true;
+						enabled_abis.remove(i);
```

Comment:
Looking back I think modifying enabled_abis directly can be quite dangerous. As enabled_abis will be empty afterwards and this will cause errors if future code contributors use it.
I could create a copy of enabled_abis in a local scope and use it to do the check. After the scope the copy will be destroyed. What do you (Godot devs) think?

Reply:
@SkyLucilfer using a local scope copy would be safer and would also help with clarifying the intent of the logic since given the current variable name, I had to double check to make sure I understood why you were doing.